controller_manager/test/test_hardware_resources/test_actuator_hardware.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 ros2_control Development Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+
+#include "./test_actuator_hardware.hpp"
+
+hardware_interface::return_type
+TestActuatorHardware::configure(const hardware_interface::HardwareInfo & actuator_info)
+{
+  fprintf(stderr, "configuring plugin with name %s\n", actuator_info.name.c_str());
```

Comment:
It's good to be consistent with printing. Why is this PR introducing *yet another* type of printing, especially it being C?

The least we could do is something along the lines of
`std::cerr << "configuring plugin with name " << actuator_info.name << std::endl;` 

but I'd still prefer to see some ROS prints

Reply:
I'll just remove the prints. 
Across the ROS2 land, we're using `fprintf` because it's thread-safe (it's also perfectly fine within C++ btw).
I won't use any ROS prints here as this shouldn't have any `rclcpp` dependencies.

Reply:
Thanks for the info. I know C functionality is perfectly fine in C++, I was mostly wondering about the discrepancy in style. On the one hand we have C++14 features, on the other we are still using C but I guess there's no dividing line here, anything that works has some leverage. I agree rclcpp may be a bit of a stretch to add.